packages/anima-cli/src/commands/sync.ts

```diff
@@ -28,224 +27,163 @@ export const builder: CommandBuilder = (yargs) =>
   yargs
     .options({
       token: { type: 'string', alias: 't' },
-      directory: { type: 'string', alias: 'd' },
+      storybook: { type: 'string', alias: 's' },
       basePath: { type: 'string', alias: 'b' },
       designTokens: { type: 'string' },
```

Comment:
since we don't use `--directory` anymore, we can reserve `-d` for "design tokens"
```suggestion
      designTokens: { type: 'string', alias: 'd' },
```